### ISSUE TITLE
Precision loss warning from thumbnails

### ISSUE THREAD

**Issue by kevinyamauchi** [napari/napari#349]
## 🐛 Bug
Converting image to uint8 for the thumbnails raises a precision loss warning:

```python
/Users/kevin.yamauchi/Documents/MyNapari/napari/.venv/lib/python3.6/site-packages/skimage/util/dtype.py:135: UserWarning: Possible precision loss when converting from float64 to uint8
  .format(dtypeobj_in, dtypeobj_out))
```

## To Reproduce

1. Run `/examples/add_image.py`
2. The following warning is printed to the terminal
```python
/Users/kevin.yamauchi/Documents/MyNapari/napari/.venv/lib/python3.6/site-packages/skimage/util/dtype.py:135: UserWarning: Possible precision loss when converting from float64 to uint8
  .format(dtypeobj_in, dtypeobj_out))
```

## Expected behavior
The warning should be suppressed

## Environment

 - napari Version (latest commit): 244c818
 - OS (e.g., Linux): OS X
 - Python version: 3.6

**Comment by kevinyamauchi** [comment]
It looks like this will be resolved by https://github.com/scikit-image/scikit-image/pull/3575. @jni, do you know if this will be merged soon?

Otherwise, we can just catch/ignore it with `warnings`.

**Comment by jni** [comment]
@kevinyamauchi I'll make another push on it.

**Comment by kevinyamauchi** [comment]
Awesome! Thanks, @jni!

**Comment by jni** [comment]
It's merged! =D If you start working with skimage master you won't see the warnings anymore. Unfortunately it'll probably be a couple of months before 0.16 is out though.

**Comment by kevinyamauchi** [comment]
Amazing! Thanks for the quick turnaround, @jni!

**Comment by sofroniewn** [comment]
This is fixed in #370, and I'm closing the issue, though once skimage `0.16` comes out we could remove our warnings catch
